src/solver/newton/newton.hpp

```diff
@@ -15,6 +15,10 @@
  * \brief Implementation of Newton method for solving system of non-linear equations
  */
 
+#if defined(_OPENACC) && !defined(DISABLE_OPENACC)
```

Comment:
Just a note: this will be also used in cuda backend. So this could be below?
```suggestion
#if (defined(__CUDACC__) || defined(_OPENACC)) && !defined(DISABLE_OPENACC)
```

Reply:
From within CUDA kernels, we can call immediately this eigen function and therefore no need to call the wrapper

Reply:
Oh ok. Perfect!